File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/android/src/main/kotlin/com/idosmart/model/IDOStressValModel.kt
//
//  IDOStressValModel.kt
//  protocol_channel
//
//  Created by hedongyang on 2023/10/30.
//

package com.idosmart.model

import com.google.gson.GsonBuilder
import com.google.gson.annotations.SerializedName

///
/// Get stress value event number

open class IDOStressValModel(stressVal: Int, threshold: Int) : IDOBaseModel {

    /// Stress value
    @SerializedName("stress_val")
    var stressVal: Int = stressVal

    /// Threshold
    @SerializedName("threshold")
    var threshold: Int = threshold


    

    override fun toJsonString(): String {
        return GsonBuilder().create().toJson(this).toString()
    }
}
    

File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/ios/Classes/Model/IDOStressValModel.swift
//
//  IDOStressValModel.swift
//  protocol_channel
//
//  Created by hc on 2023/10/12.
//

import Foundation
/// Get stress value event number
@objcMembers
public class IDOStressValModel: NSObject, IDOBaseModel {
    /// Stress value
    public var stressVal: Int
    /// Threshold
    public var threshold: Int

    enum CodingKeys: String, CodingKey {
        case stressVal = "stress_val"
        case threshold = "threshold"
    }

    public init(stressVal: Int, threshold: Int) {
        self.stressVal = stressVal
        self.threshold = threshold
    }

    public func toJsonString() -> String? {
        if let jsonData = try? JSONEncoder().encode(self) {
            let jsonString = String(data: jsonData, encoding: .utf8)
            return jsonString
        }
        return nil
    }
}
